cdk/DevPipeline.py

```diff
@@ -0,0 +1,56 @@
+from aws_cdk import (
+    core
+)
+from aws_cdk.pipelines import CodePipeline, CodePipelineSource, ShellStep, ManualApprovalStep
+
+from makerspace import MakerspaceStage
+from accounts_config import accounts
+
+class DevPipeline(core.Stack):
+    def __init__(self, app: core.App, id: str, *,
+                 env: core.Environment) -> None:
+        super().__init__(app, id, env=env)
+
+        deploy_cdk_shell_step = ShellStep("Synth",
+            input=CodePipelineSource.connection("Mjtlittle/unified-makerspace", "pipeline-development",
```

Comment:
Just to be clear- this is just for feedback, and wont be pushed, correct?

Reply:
correct! I plan on revising the PR tonight to work with the actual pipeline

Reply:
Sweet!